R/readMAgPIE.R

```diff
@@ -70,34 +70,27 @@ readMAgPIE <- function(subtype) {
     }
 
   } else if (subtype == "supplyCurve_magpie_40") {
-    regcode <- "690d3718e151be1b450b394c1064b1c5"
+    regcode <- ""
```

Comment:
why don't we need a regionscode?

Reply:
I will include it again in the next update, which will probably come soon.